src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -90,7 +90,7 @@ module sonic-device_metadata {
                 leaf type {
                     type string {
                         length 1..255;
-                        pattern "ToRRouter|LeafRouter|SpineChassisFrontendRouter|ChassisBackendRouter|ASIC";
+                        pattern "ToRRouter|LeafRouter|SpineChassisFrontendRouter|ChassisBackendRouter|ASIC|MgmtToRRouter|SpineRouter|BackEndToRRouter|BackEndLeafRouter";
```

Comment:
Should we also add EPMS?

Reply:
Good catch Ying! I have added that. Also MgmtTsToR